app/code/Magento/Backend/Test/Mftf/ActionGroup/DeleteEntitySecondaryGridActionGroup.xml

```diff
@@ -10,7 +10,7 @@
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
     <!-- Action group to delete an item given that items name -->
     <!-- Must already be on the admin page containing the grid -->
-    <actionGroup name="deleteEntitySecondaryGrid">
+    <actionGroup name="DeleteEntitySecondaryGridActionGroup">
```

Comment:
I would like to clarify this part. It looks like a backward-incompatible change in case if some 3rd-party extension uses this action group for MFTF tests. Or I may miss something

Reply:
In worst-case-scenario we would be able to merge that only to `2.5-develop`.

Reply:
Fingers crossed :)